pandas/tests/io/test_excel.py

```diff
@@ -70,7 +70,6 @@ def get_csv_refdf(self, basename):
     def get_excelfile(self, basename, ext):
         """
         Return test data ExcelFile instance. Test data path is defined by
-        pandas.util.testing.get_data_path()
```

Comment:
I think you need to remove the beginning of the sentence on the previous line as well?